aim/cli/up/commands.py

```diff
@@ -75,7 +76,10 @@ def up(dev, host, port, workers, uds,
     if repo_status == RepoStatus.MISSING:
         init_repo = None
         if not force_init:
```

Comment:
As we have a dedicated `force_init` flag, does it make sense to check the `--yes` here?

Reply:
@alberttorosyan sorry for the late reply, I didn't notice the `force_init` flag, will add a commit to fix this now